Add note about second install upgrade might help (#2423)
PostHog/posthog.com: contents/docs/self-host/deploy/snippets/installing.mdx

helm repo update
helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
```

<details>
<summary>Troubleshooting</summary>

> If you don't see some pods come up (e.g. `chi-posthog-posthog-0-0-0` pod is missing after deleting a previous instance and trying to install PostHog again), try running `helm upgrade -f values.yaml -n posthog posthog posthog/posthog`.

Reviewer: Why the clickhouse pod should be missing?

Author: no idea, it just happens often - haven't had time to figure out why yet.

Author: ```suggestion
If you don't see some pods or services come up (e.g. `chi-posthog-posthog-0-0-0` pod or `clickhouse-posthog` service is missing), try running `helm upgrade -f values.yaml -n posthog posthog posthog/posthog`.
```
